openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -351,20 +360,24 @@ def _objfunc(self, dv_dict):
         """
         model = self._problem.model
         fail = 0
+        metadata = self.metadata
 
         try:
             for name in self._indep_list:
                 self.set_design_var(name, dv_dict[name])
 
-            # print("Setting DV")
```

Comment:
I'd really like my debug statements to stay in until we are fully stable. I use them all the time.